testing/jitterentropy-rngd/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Henrik Riomar <henrik.riomar@gmail.com>
+# Maintainer:
+pkgname=jitterentropy-rngd
+pkgver=1.0.8
+pkgrel=0
+pkgdesc="CPU Jitter RNG Daemon"
+url="http://www.chronox.de/jent.html"
+arch="all"
+license="GPL-2.0 BSD"
```

Comment:
BSD isn't spdx and is also ambiguous

GPL-2.0 needs -or-later or -only suffix as per SPDX 3.0

Reply:
thanks, is `"GPL-2.0-only BSD-3-Clause"` OK? when its GPL or BSD (not GPL+BSD)

Reply:
check the source code that are under GPL-2.0- , they will have a 

`this is licensed under GNU GPL 2, **or later**` or `this is licensed under GNU GPL 2 **only**`

Reply:
there is no later for gpl in the sources.

My question is, whats the proper form for license, when all files are dual licensed. I.e. its not some files bsd and some gpl. But all are dual.

Reply:
I changed to "GPL-2.0-only BSD-3-Clause"